frontend/src/metabase/collections/components/ActionMenu/ActionMenu.unit.spec.tsx

```diff
@@ -1,78 +1,98 @@
 import React from "react";
-import { render, screen } from "@testing-library/react";
+import { screen } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
+import { renderWithProviders } from "__support__/ui";
+import { Collection, CollectionItem } from "metabase-types/api";
 import {
   createMockCollection,
   createMockCollectionItem,
 } from "metabase-types/api/mocks";
-import ActionMenu, { ActionMenuProps } from "./ActionMenu";
+import ActionMenu from "./ActionMenu";
+
+interface SetupOpts {
+  item: CollectionItem;
+  collection?: Collection;
+}
+
+const setup = ({
+  item,
+  collection = createMockCollection({ can_write: true }),
+}: SetupOpts) => {
+  const onCopy = jest.fn();
+  const onMove = jest.fn();
+
+  renderWithProviders(
+    <ActionMenu
+      item={item}
+      collection={collection}
+      onCopy={onCopy}
+      onMove={onMove}
+    />,
+  );
+
+  return { onCopy, onMove };
```

Comment:
We've never seem to use these return values, should we still return them? This gives me the impression that they'll be asserted somewhere else.

Reply:
We use them in one of the tests

Reply:
I couldn't find we use the return value anywhere in this file https://github.com/metabase/metabase/blob/f9f71d85a4e9aa249da33cc8fb8c095265c39fb3/frontend/src/metabase/collections/components/ActionMenu/ActionMenu.unit.spec.tsx. What test were you mentioning?

Reply:
Forgot to push the test, fixed

Reply:
That's good, but only the `onMove` is used actually. Should we not return `onCopy` then? Or at least test `onCopy` which is actually a duplicate button (I was thinking when copying you'll need to paste it yourself, that's another story, I wish it's called `onDuplicate` instead)